bin/common/common_lib.sh

```diff
@@ -60,7 +60,7 @@ function env_check_info() {
 }
 
 function dir_check_err() {
-    if [[ -d ${!1} ]]
+    if [[ ! -d ${!1} ]]
```

Comment:
Thanks very much for catching this! It is indeed incorrectly coded. However, the correction you made is still incorrect - after testing to be certan, it should be:

`if [[ ! -d ${1} ]]`

Reply:
No because ${!1} is use to dereference the variable name pass in the argument

Reply:
My apologies, I was wrong. After revisiting the test I had originally used I found I didn't use variable names to test with but rather hardcoded directory paths. Re-tested using variable names and it is indeed correct. @jmccormick2001 this is good to go